chapters/equations.tex

```diff
@@ -926,23 +926,17 @@ \subsection{The Number of Equations Needed for Initialization}\label{the-number-
 diagnostics are given.
 \end{example}
 
-\subsection{Recommended selection of start-values}\label{recommended-selection-of-start-values}
+\subsection{Recommended selection of start-attributes}\label{recommended-selection-of-start-attributes}
```

Comment:
I prefer to keep start-values here, as the goal is to select which values to start at. Technically you do it by selecting attributes for some variables, but the _intention_ is to select the starting values for variables.
```suggestion
\subsection{Recommended selection of start-values}\label{recommended-selection-of-start-values}
```

Reply:
OK, no big deal.  I did it almost as suggested, removing the hyphen in "start-values".  Resolving.